docker/fastqc/1.0.0-alpha/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM stjudecloud/conda-base:1.0.0-alpha AS builder
+
+RUN conda create -n fastqc \
+        fastqc==0.11.8 \
+        -y \
+    && conda clean --all
+
+FROM debian:10-slim
+COPY --from=builder /opt/conda/envs/fastqc/bin/ /opt/conda/envs/fastqc/bin/
```

Comment:
Could this be one `COPY` for all of the `fastqc` dir to make the Dockerfile cleaner?

Reply:
There a bunch of directories in there that aren't needed. I haven't checked their size, so might not actually impact the final docker image size significantly. But this is the most space efficient method.

Reply:
Okay, sounds okay to me then!